packages/runtime/src/firebase/remoteConfigValuesDefaults.ts

```diff
@@ -8,23 +12,44 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   sentryNetworkErrors: string
   dappListApiUrl: string
   celoNews: string
-} = {
-  inviteRewardsVersion: 'none',
-  walletConnectV2Enabled: true,
-  pincodeUseExpandedBlocklist: false,
-  logPhoneNumberTypeEnabled: false,
-  allowOtaTranslations: false,
-  sentryTracesSampleRate: DEFAULT_SENTRY_TRACES_SAMPLE_RATE,
-  sentryNetworkErrors: DEFAULT_SENTRY_NETWORK_ERRORS.join(','),
-  maxNumRecentDapps: 0,
-  dappsWebViewEnabled: false,
-  dappListApiUrl: '',
-  fiatConnectCashInEnabled: false,
-  fiatConnectCashOutEnabled: false,
-  coinbasePayEnabled: false,
-  showSwapMenuInDrawerMenu: false,
-  maxSwapSlippagePercentage: 2,
-  networkTimeoutSeconds: 30,
-  celoNews: JSON.stringify({} as RemoteConfigValues['celoNews']),
-  priceImpactWarningThreshold: 0.04,
-}
+} = isE2EEnv
```

Comment:
It looks like some values are the same between these two configs. Should they be extracted to a standard config used by both?

Reply:
Yes good point, I've just inlined the `isE2eEnv` checks to avoid repeated properties and more easily see differences.